### ISSUE TITLE
Using tests/requirements.txt in GitHub CI fails

### ISSUE THREAD

**Issue by sthiele** [MiniZinc/libminizinc#560]
[GitHub no longer allows using the unencrypted Git protocol. ](https://github.blog/2021-09-01-improving-git-protocol-security-github/)
Therefore, now setting up test in the CI via   `pip3 install -r libminizinc/tests/requirements.txt` fails

```
  pip3 install -r libminizinc/tests/requirements.txt
  shell: /usr/bin/bash -e {0}
Collecting PyYAML==5.4
  Downloading PyYAML-5.4-cp38-cp38-manylinux1_x86_64.whl (662 kB)
Collecting pytest==6.*
  Downloading pytest-6.2.5-py3-none-any.whl (280 kB)
Collecting py==1.9.*
  Downloading py-1.9.0-py2.py3-none-any.whl (99 kB)
Collecting pytest-html==2.*
  Downloading pytest_html-2.1.1-py2.py3-none-any.whl (16 kB)
Collecting pytest-xdist==2.*
  Downloading pytest_xdist-2.5.0-py3-none-any.whl (41 kB)
Collecting pytest-instafail==0.4.*
  Downloading pytest_instafail-0.4.2-py2.py3-none-any.whl (4.2 kB)
Collecting minizinc
  Cloning git://github.com/MiniZinc/minizinc-python (to revision develop) to /tmp/pip-install-rnyo26w7/minizinc
  Running command git clone -q git://github.com/MiniZinc/minizinc-python /tmp/pip-install-rnyo26w7/minizinc
  fatal: remote error:
    The unauthenticated git protocol on port 9418 is no longer supported.
  Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
ERROR: Command errored out with exit status 128: git clone -q git://github.com/MiniZinc/minizinc-python /tmp/pip-install-rnyo26w7/minizinc Check the logs for full command output.
Error: Process completed with exit code 1.

**Comment by cyderize** [comment]
Hi, this has already been fixed in the develop branch (dfc5757f1c18fcbda31dea2306392f9a3edba4e4)

**Comment by sthiele** [comment]
Awesome, thanks.
